src/js/components/List/List.js

```diff
@@ -192,7 +193,14 @@ const List = React.forwardRef(
                     adjustedEvent.item = item;
                     adjustedEvent.index = index;
                     onClickItem(adjustedEvent);
+                    // put focus back on the List container to meet
+                    // WCAG accessibility guidelines that focus remains
+                    // on `ul`
+                    (ref || listRef).current.focus();
                   },
+                  // prevent default focus outline from being shown
+                  // onMouseDown since `ul` is maintaining focus
+                  onMouseDown: event => event.preventDefault(),
```

Comment:
Do we need this? It feels like we might be preventing behavior that might be wanted somewhere.

Reply:
If this is not here, the `li` tries to grab focus, I wasn't able to find another way (to your below comment) to intercept this element from receiving focus. However, this does not have an effect on the `onClick`.

![focus](https://user-images.githubusercontent.com/12522275/88981224-6032b380-d27a-11ea-976c-ad5436fa59b5.gif)